docs/site/package.json

```diff
@@ -13,21 +13,21 @@
     "clean": "rimraf build"
   },
   "devDependencies": {
-    "babel-cli": "^6.18.0",
+    "babel-cli": "^6.22.2",
```

Comment:
> That not what ^ does.

That impact us when running `yarn upgrade` that also impact users when installing the lib.